default.nix

```diff
@@ -160,8 +182,30 @@ in
         };
       };
     } // (builtins.foldl' (x: y: x // y) { } (builtins.genList
+      (index: {
+        "accentor-worker-${toString (index)}" = {
+          after = [ "network.target" "accentor-api.service" "postgresql.service" ];
+          requires = [ "postgresql.service" ];
+          wantedBy = [ "multi-user.target" ];
+          environment = env // {
+            GOOD_JOB_QUEUES = (builtins.elemAt cfg.workers index);
+          };
+          path = [ pkgs.ffmpeg gems gems.wrappedRuby ];
+          serviceConfig = {
+            EnvironmentFile = cfg.environmentFile;
+            Type = "simple";
+            User = "accentor";
+            Group = "accentor";
+            Restart = "on-failure";
+            WorkingDirectory = api;
+            ExecStart = "${gems}/bin/bundle exec good_job start";
+          };
+        };
+      })
+      (builtins.length cfg.workers)))
```

Comment:
Mapping the list instead of using `genList`, `length` and `elemAt` seems a bit more natural to me.

Reply:
Duh - I have a variation on this in several projects and somehow never thought of this :)